src/dso_api/dynamic_api/serializers.py

```diff
@@ -685,7 +722,7 @@ def _temporal_link_serializer_factory(
     using the generic `DSORelatedLinkField` field.
     """
     if not related_model.is_temporal():
-        raise ValueError("Use DSORelatedLinkField instead")
+        raise ValueError("Use _nontemporal_link_serializer_factory in stead")
```

Comment:
To ensure PyCharm and other editor can refactor this fearlessly without having to depend on textual similarities, please use:
```python
f"Use {_nontemporal_link_serializer_factory.__name__} instead"
```